community/cypher/frontend-3.1/src/main/scala/org/neo4j/cypher/internal/frontend/v3_1/ast/CaseExpression.scala

```diff
@@ -37,3 +39,9 @@ case class CaseExpression(expression: Option[Expression], alternatives: Seq[(Exp
     } chain this.specifyType(possibleTypes)
   }
 }
+
+object CaseExpression {
+  def apply(expression: Option[Expression], alternatives: List[(Expression, Expression)], default: Option[Expression])(position: InputPosition):CaseExpression =
```

Comment:
It is but parboiled creates Lists so I need this extra constructor to turn it into an IndexedSeq